src/nems_subscription_create/external_integrations/pds/api_client.py

```diff
@@ -73,17 +72,32 @@ def _get_valid_access_token(self) -> str:
             self._access_token_expires_at is not None
             and self._access_token_expires_at > datetime.utcnow()
         ):
+            _LOGGER.info("Reusing valid access token")
             return self._access_token
 
+        _LOGGER.info(
```

Comment:
@michal-kras I assume this was for debug purposes to detect the bug you were talking about - can you remove? (we defo shouldn't be logging our secret access token)